ports/memory/common.mk

```diff
@@ -51,12 +51,16 @@ MAKE_ARGS ?= -j $(firstword $(JLEVEL) 1)
 ifndef NO_TARGET_OVERRIDE
 memory_all:
 	@mkdir -p build
+	@if ! grep -q QNX "$(QNX_PROJECT_ROOT)/test/CMakeLists.txt" ; then echo "if(QNX)" >> $(QNX_PROJECT_ROOT)/test/CMakeLists.txt; echo "install(TARGETS foonathan_memory_test DESTINATION bin/) #QNX" >> $(QNX_PROJECT_ROOT)/test/CMakeLists.txt; echo "endif() #QNX" >> $(QNX_PROJECT_ROOT)/test/CMakeLists.txt; fi
```

Comment:
So obviously this isn't the cleanest way to do this; would it be better to generate/apply a patch file? I'm not super aware of how resilient those are i.e. if they add more lines to cmakelists.txt in the future, would the patch break?